### PR TITLE
Chapter 18: Pipes commit to repo

### PR DESCRIPTION
**Pull request by rmetcalf414**
Jon, I'm unclear the first two commits I have listed below. The third is the updated slide deck.
Thank you!

### CODE REVIEW

File: 18-pipes.Rmd [hunk 1]
```diff
@@ -2,13 +2,136 @@
...
+One method of wrangling data is to just overwrite and reassign a calculation to its original object. But, there is a problem...
+
+1. Overwriting an object prevents clean debugging. You'll have to rerun your code over and over to see what is changing.
+2. The repetition of the object being transformed implies... "what does my data look like now?"
+
+If you want to make yourself crazy... continually read in a new copy of your data just to walk through the process over agian. This is *not* conducive to efficiency.
+
+## The pipe makes lexical transformations
+
+- `magrittr` reassembles the code in the pipe to a form that works by overwriting an intermediate object, see [link]()
```
Comment (jonthegeek) [inline 18-pipes.Rmd:74]: Where is this link supposed to point?

**Review by jonthegeek — CHANGES_REQUESTED** [review]
I made some small tweaks. Also see the comment about the missing link on line 74.

File: 18-pipes.Rmd (hunk 1 above)
Comment (rmetcalf414) [reply 18-pipes.Rmd:74]: Shoot! I was off on a tangent to find Lexical Transformation/Scoping....I hadn't finished my thought (with the link!). Referencing Chapter 6 of Advanced R. https://adv-r.hadley.nz/functions.html#lexical-scoping. I'll make the change and resubmit.

**Review by jonthegeek — APPROVED** [review]
See ya shortly!

**Pull request merged**